systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/RollingUpdateST.java

```diff
@@ -624,7 +624,7 @@ void testClusterOperatorFinishAllRollingUpdates(ExtensionContext extensionContex
 
         TestUtils.waitFor("rolling update starts", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_STATUS_TIMEOUT,
             () -> kubeClient(namespace).listPods(namespace).stream().filter(pod -> pod.getStatus().getPhase().equals("Running"))
-                    .map(pod -> pod.getStatus().getPhase()).collect(Collectors.toList()).size() < kubeClient().listPods().size());
+                    .map(pod -> pod.getStatus().getPhase()).collect(Collectors.toList()).size() < kubeClient().listPods(namespace).size());
```

Comment:
This is already done here -> https://github.com/strimzi/strimzi-kafka-operator/pull/6631.

Reply:
Okay, good catch! Thanks :)